starknet_py/net/account/account.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         client: Client,
         signer: Optional[BaseSigner] = None,
         key_pair: Optional[KeyPair] = None,
-        chain: Optional[StarknetChainId] = None,
+        chain: Optional[Union[StarknetChainId, str, int]] = None,
```

Comment:
nit: For readability we can extract this type to something like `ChainId = Union[StarknetChainId, str, int]`, then it cal be also used it in `StarkCurveSigner`

Reply:
the problem is that StarkCurveSigner `chain_id: Union[StarknetChainId, int]` could be simplified to `chain_id: int` but I assume that we want to explicitly show StarknetChainId

Reply:
In that case let's leave `chain_id: Union[StarknetChainId, int]` in `StarkCurveSigner`.